packages/amplify-e2e-tests/src/cleanup-e2e-resources.ts

```diff
@@ -665,6 +724,8 @@ const cleanupAccount = async (account: AWSAccountInfo, accountIndex: number, fil
   const appPromises = AWS_REGIONS_TO_RUN_TESTS.map(region => getAmplifyApps(account, region));
   const stackPromises = AWS_REGIONS_TO_RUN_TESTS.map(region => getStacks(account, region));
   const bucketPromise = getS3Buckets(account);
+  // include 'us-east-1' due to custom regional mapping for pinpoint resource: https://github.com/aws-amplify/amplify-cli/blob/main/packages/amplify-provider-awscloudformation/src/aws-utils/aws-pinpoint.js#L49-L74
+  const orphanPinpointApplicationsPromise = ['us-east-1', ...AWS_REGIONS_TO_RUN_TESTS].map(region => getOrphanPinpointApplications(account, region));
```

Comment:
why can't we just add us-east-1 to AWS_REGIONS_TO_RUN_TESTS ?

Reply:
@awsluja what do you think?

Reply:
not sure if there might be some resources in the us-east-1 that are not test related, I believe we do not run our tests on 'us-east-1' and this is why the cleanup was excluding it, however for pinpoint we have custom mappings that use 'us-east-1'

Reply:
i don't see a reason why not - I think we should add us-east-1